loky/backend/resource_tracker.py

```diff
@@ -52,7 +69,8 @@
 _IGNORED_SIGNALS = (signal.SIGINT, signal.SIGTERM)
 
 _CLEANUP_FUNCS = {
-    'folder': shutil.rmtree
+    'folder': shutil.rmtree,
+    'file': os.unlink
```

Comment:
Maybe we should wrap those functions to ignore `OSError` (no such file or directory) if the resource has already been unlinked by another process.

Reply:
I agree It would make some test logs less verbose, but I'm not sure about this change yet because it is useful information to know when we overshoot the number of `maybe_cleanup` calls, especially as I am calibrating the reference counting process of shared `numpy` arrays in `joblib`.